synchronization/src/androidTestMovebis/java/de/cyface/synchronization/DataTransmissionTest.java

```diff
@@ -177,17 +175,18 @@ public void updatedProgress(float percent) {
      * @param stream The stream to print the MD5 sum for.
      * @throws IOException Thrown if the stream is not readable.
      * @throws NoSuchAlgorithmException Thrown if MD5 Algorithm is not supported
+     *             /
+     *             private void printMD5(final @NonNull InputStream stream) throws IOException, NoSuchAlgorithmException
+     *             {
+     *             MessageDigest md = MessageDigest.getInstance("MD5");
+     *             byte[] content = new byte[stream.available()];
+     *             // noinspection ResultOfMethodCallIgnored - because we don't care
+     *             stream.read(content);
+     *             byte[] theDigest = md.digest(content);
+     *             StringBuilder sb = new StringBuilder(theDigest.length * 2);
+     *             for (byte b : theDigest)
+     *             sb.append(String.format("%02x", b));
+     *             Log.i(TAG, sb.toString());
+     *             }
```

Comment:
or should I just delete this?

Reply:
probably. this was just to check against something.